nextflow_schema.json

```diff
@@ -48,24 +48,34 @@
                 },
                 "min_counts": {
                     "type": "integer",
-                    "description": "Minimum number of variant observations required.",
+                    "description": "minimum counts for variant to be recognized. All variants below min_counts will be set to 0",
                     "minimum": 1,
                     "default": 3
                 },
+		"sliding_window_size": {
+                    "type": "integer",
+                    "description": "To flatten graphs in plots (e.g. GLOBAL_POS_BIASES_COUNTS function)",
+                    "default": 10
+                },
+		"aimed_cov": {
+                    "type": "integer",
+                    "description": "aimed coverage (assuming equal spread) to visualize threshold in plots",
+                    "default": 100
```

Comment:
one can set minimum and maximum values in json schema: https://json-schema.org/understanding-json-schema/reference/numeric#range , to help with some parameter validation